generators/CoreCLRStatefulActor/index.js

```diff
@@ -140,18 +160,20 @@ var ClassGenerator = generators.Base.extend({
     }
 
     this.fs.copyTpl(
-      this.templatePath('service/app/appPackage/servicePackage/Code/entryPoint.sh'),
-      this.destinationPath(path.join(appPackage , appPackagePath, servicePackage, 'Code', 'entryPoint.sh')),
+      this.templatePath('service/app/appPackage/servicePackage/Code/entryPoint'+extension2),
+      this.destinationPath(path.join(appPackage , appPackagePath, servicePackage, 'Code', 'entryPoint'+extension2)),
```

Comment:
Are we still using entrypoint script for windows?

Reply:
No we are not using. We were generating it but not actually using it . Now its removed.